demo/schemas/user-collection.js

```diff
@@ -7,6 +7,17 @@ module.exports = {
     description: 'A collection of users.',
     type: 'object',
     properties: {
+        limit: {
```

Comment:
So we just add them to the schema and the definition of the route's URL above having these two property names as query string parameters in it maps them as query string parameters and not standard resource properties?